Library/Homebrew/cmd/help.rb

```diff
@@ -74,6 +74,12 @@ def help(cmd = nil, flags = {})
   end
 
   def command_help(path)
+    if Pathname(path).fnmatch?("*/Homebrew/cmd/*.rb")
```

Comment:
Hi @apjanke, it already does this inside this if condition:
`return command_help_cmd(class_name) if Homebrew.const_defined?(class_name)`